style.css

```diff
@@ -1,3 +1,4 @@
+@charset "UTF-8";
```

Comment:
@EvanHerman This looks like a regression of a bug we fixed a while back in https://github.com/godaddy/wp-primer-theme/commit/f0da8e4a3021218926675b6e3280439c8970c634. Looks like we need to understand the root cause and fix it.

Reply:
@fjarrett Looks like `replace:charset` runs on default `grunt` task, but not on `grunt:watch`